spec/docs/logicalSource.md

```diff
@@ -1,15 +1,17 @@
-# Defining Logical Sources
+# Defining Logical Iterables and Logical Sources
 
-A <dfn>logical source</dfn> is an abstract construct to describe data access and iteration for a [=data source=] such that it can be mapped to [=RDF triples=].
+A <dfn>logical iterable</dfn> is an abstract construct to describe data access and iteration for a [=data source=].
 
-A [=logical source=] (`rml:LogicalSource`) MUST have:
+A [=logical iterable=] (`rml:LogicalIterable`) MUST have:
 * exactly one `rml:referenceFormulation` property, whose value is a <dfn>reference formulation</dfn> which defines how the underlying [=data source=] is to be accessed, and which [=expressions=] can be evaluated on [=logical iterations=],
```

Comment:
A field can have an implicit reference formulation (the same reference formulation as its parent field). Therefore it is possible that the `rml:referenceFormulation` property is missing. I suggest 'zero or one', similar to the `rml:iterator` property.

Reply:
agreed, will fix